charts/thoras/values.yaml

```diff
@@ -27,7 +27,9 @@ metricsCollector:
   persistence:
     enabled: false
     volumeName: ""
-    storageClassName: ""
+    storageClassName: "efs-sc-thoras"
```

Comment:
I made this a sane default since users may now be using auto-create-sc functionality of the chart